Tasks/DownloadPackageV0/make.json

```diff
@@ -0,0 +1,9 @@
+{
+    "rm": [
+        {
+            "items": [
+                "node_modules/azure-pipelines-tasks-packaging-common-v3/node_modules/azure-pipelines-task-lib"
```

Comment:
why is it needed to rm azure-pipelines-task-lib?

Reply:
We update task-lib in the package.json to the new version. But we are not yet updating azure-pipelines-tasks-packaging-common-v3, which has a dependency on previous version of task-lib. And therefore, the task will contain two versions of task-lib. But the problem is that task-lib is written as a singleton, and therefore it initializes only once, with one of the versions, making it impossible to use two versions simultaneously. So, we remove the dependency of old task-lib. In this case we are having only one version of task-lib in the task.